packages/core/contracts/cross-chain-oracle/test/GovernorHubMock.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../interfaces/ParentMessengerInterface.sol";
+
+contract GovernorHubMock {
```

Comment:
Why do you need this mock? It seems like you can just use the GovernorHub but with one small change to setting messenger per chain ID instead of 1 messenger for all chain IDs

Reply:
I added this because the actual GovernorHub does not contain all the event data that I needed. I can however refactor the tests to work without this and delete the mock.

Reply:
I ended up removing all three mocks. it made the tests a bit more complex but it uses the actual contracts that would be used in production.